### PR TITLE
[session 2] 이수빈 과제제출합니다. 

### PR DESCRIPTION
**Pull request by Soovni**
시각화, 전처리 과제 제출합니다! 
평소 관심있던 시각화와 전처리에 대해 더 자세히 알게 되어 좋습니다!
통계마크다운 정리를 하며, 다음시간 통계시간에 배울 내용이 기대된다는 생각이 들었습니다.

### CODE REVIEW

**Comment by sangho24** [comment]
[시각화]

(1) 승선항(Embarked)별 생존여부를 hue파라미터를 이용해 확인
f, ax = plt.subplots(1, figsize=(10,6))

이 코드는 한 화면에 여러 개의 그래프를 띄울 때 사용하는데, 지금은 시각화할 그래프가 하나이므로 굳이 이 코드를 사용하지 않으셔도 됩니다.
sns.countplot(data=data, x = 'Embarked', hue = 'Survived')
이것만 써주시면 됩니다!

(4) Pclass 별 생존율을 barplot을 통해 확인해보세요.
생존율을 따로 구해서 시각화하는 방법도 있겠으나, barplot을 이용하면 알아서 비율을 계산해서 시각화해줍니다. 나중에 따로 그려보세요 :)
sns.barplot(data=data, x='Pclass', y='Survived')

(10) 제시해주신 내용은 맞습니다만, 코드가 작성되어 있지 않습니다 ㅜㅜ 

[전처리]

df['normalized-losses'] = df['normalized-losses'].astype(float)
avg_norm_loss = df['normalized-losses'].mean()
-> avg_norm_loss=df["normalized-losses"].astype(float).mean()
. 연산자 뒤에는 일반적으로 여러 개 붙이는 것이 가능해서, 한 줄로 쓸 수 있습니다.


df['price'].info()  #info 가 안되는데 왜일까요?
-> 일단 오류메세지는 Series 객체는 info 메서드가 없다고 하는데, 다른 분들은 똑같이 했을 때 문제가 없었던 걸로 봐선 이전에 깔았던 판다스 버전 문제일 수도 있을 거 같습니다... 저번 실습때도 이런 에러가 많던데, 파이썬이나 모듈을 여러 번 깔았다면 버전 최신화하는 것도 좋아보입니다! 
일단 대체할 수 있는 방법으로는 [df.info](http://df.info/)() 를 찍던, df.describe() 등이 있을 것 같아요.

[사전과제]
주피터 노트북에서 마크다운으로 정리해주셔야 해요..! 오늘 알려드릴게요. 과제하느라 고생했어요!
